docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -584,6 +584,7 @@ COPY --chown=quarkus:quarkus .mvn /code/.mvn
 COPY --chown=quarkus:quarkus pom.xml /code/
 USER quarkus
 WORKDIR /code
+RUN chmod +x ./mvnw
```

Comment:
Not very familiar with the Docker format but can't we copy and keep the permissions above?

Reply:
Good question. I am also not an expert!  let me find out

Reply:
Combined them into one command